### ISSUE TITLE
Background transparency

### ISSUE THREAD

**Issue by LoganDark** [sagb/alttab#137]
`alttab` could support RRGGBBAA syntax for transparency: `#000000aa` or something

or something else, but I would really like the option to have a transparent background so I could have my compositor do a background blur.

Thanks!

**Comment by sagb** [comment]
Probably a nice feature. I have no time for this, patches are welcomed.

**Comment by sagb** [comment]
Probably a compositor can do everything on its own:
https://confluence.jaytaala.com/display/TKB/My+Manjaro+i3+setup#MyManjaroi3setup-Ricingalttab

**Comment by LoganDark** [comment]
Yes, it's possible for a compositor to implement it. Maybe I'll take a stab at making my own once I can get my new Linux install to work.
